### PR TITLE
Remove jomwells/ambihue

### PR DESCRIPTION
**Pull request by Mr-Groch**
jomwells/ambihue is not maintained and soesn't work with newest HA. I've also changed Ambilight+Hue logic, to use HueLamp api instead of using menu hack

### CODE REVIEW

**Comment by Mr-Groch** [comment]
Brand icon is awaiting to be merged:

https://github.com/home-assistant/brands/pull/2578

**Review by ludeeus — CHANGES_REQUESTED** [review]
This needs to be clit in 2 PR's

**Comment by Mr-Groch** [comment]
Done, this PR is for removing jomwells/ambihue, and new PR created for adding replacement: #985

File: removed [hunk 1]
```diff
@@ -495,5 +495,10 @@
     "reason": "Repository does not exist",
     "removal_type": "remove",
     "link": "https://github.com/hacs/integration/issues/2080"
+  },
+  {
+    "repository": "jomwells/ambihue",
+    "reason": "It's abandoned",
+    "removal_type": "replaced"
```
Comment (ludeeus) [inline removed:502]: Missing link to information about why

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: removed (hunk 1 above)
Comment (Mr-Groch) [reply removed:502]: Can it be link to this PR or to replaced repo?

Comment (ludeeus) [reply removed:502]: yes

File: removed [hunk 2]
```diff
@@ -495,5 +495,11 @@
     "reason": "Repository does not exist",
     "removal_type": "remove",
     "link": "https://github.com/hacs/integration/issues/2080"
+  },
+  {
+    "repository": "jomwells/ambihue",
+    "reason": "It's abandoned",
+    "removal_type": "replaced",
+    "link": "https://github.com/Mr-Groch/ambihue"
```
Comment (ludeeus) [inline removed:503]:
```suggestion
    "link": "https://github.com/hacs/default/pull/955"
```

**Pull request merged**